roboteam_world/src/Handler.cpp

```diff
@@ -31,6 +33,23 @@ void Handler::start(std::string visionip, std::string refereeip, int visionport,
         std::string file_name = "world_log_" + std::to_string(days) + "_" +std::to_string(hours) + "_" + std::to_string(minutes) +"_" + std::to_string(seconds) + "_" + std::to_string(mili)+".log";
         fileWriter = rtt::LogFileWriter();
         fileWriter.value().open(file_name);
+        //Create a signal handler so that when we crash data is still saved,
+        // so that the file is still closed when the program is unexpectedly terminated or an error is thrown
+        //If we do not do this, the filestream buffer may contain the messages which caused the crash,
```

Comment:
'the crash' is ambiguous. Maybe add reference to the issue that this commit solves

Reply:
For future team members

Reply:
It refers to any crash of the program, so in that sense it is indeed ambiguous. A signal will typically cause the program to crash.